### PR TITLE
Log workflow modification

### PR DESCRIPTION
**Pull request by ventilooo**

### CODE REVIEW

File: standard_plus_issue/i18n/fr.po [hunk 1]
```diff
@@ -269,12 +265,6 @@ msgid "Object"
 msgstr "Objet"
 
 #. module: standard_plus_issue
-#: code:addons/standard_plus_issue/models/standard_plus_issue.py:113
-#, python-format
-msgid "Please join a screenshot."
-msgstr "Veuillez joindre une capture d'écran."
```
Comment (JulieSFL) [inline standard_plus_issue/i18n/fr.po]: Why did that entry disappear?

File: standard_plus_issue/models/standard_plus_issue.py [hunk 2]
```diff
@@ -153,6 +154,9 @@ def action_set_submitted(
         self.ensure_one()
         self.send_by_email(self.prepare_mail_attachments(), template_ref)
         self.state = 'submitted'
+        self.message_post(
+            body=_('Email send to ' + self.property_support_email),
```
Comment (JulieSFL) [inline standard_plus_issue/models/standard_plus_issue.py]: 'Email sen**t** to'
also the trad is not present in the trad file

**Comment by JulieSFL** [comment]
What about that part?

> La réception de courriel également: par exemple, le client nous alloue une adresse courriel réservée à l'objet standard.plus.issue
Si nous répondons à ce courriel depuis notre outil de ticketing, il devrait pouvoir recevoir la réponse dans le ticket associé

**Comment by ventilooo** [comment]
It require much more work. It will be done in another PR ;)

**Comment by ventilooo** [comment]
@joaoalf @dufresnedavid ping ?

**Comment by joaoalf** [comment]
@ventilooo When you say that it will be made in another PR, when it's gonna happen?

File: standard_plus_issue/models/standard_plus_issue.py [hunk 3]
```diff
@@ -153,6 +154,9 @@ def action_set_submitted(
         self.ensure_one()
         self.send_by_email(self.prepare_mail_attachments(), template_ref)
         self.state = 'submitted'
+        self.message_post(
+            body=_('Email sent to ') + self.property_support_email,
+            subject=None, message_type='notification')
```
Comment (ghost) [inline standard_plus_issue/models/standard_plus_issue.py:159]: We will need to talk about how emails are used in this module.

You should not have post a notification. The email sent should appear inside the mail thread.
Also, the address used by the support should be a partner, not a bare email address.

When someone logs a note inside the issue, as a support team member, I want to be advised.

Comment (ventilooo) [reply standard_plus_issue/models/standard_plus_issue.py:159]: That's not in the scope of the initial  change request.
Your issues are valuable and we should put them in the next release.

**Comment by ventilooo** [comment]
@bjoliveau ping

**Review by rimbendhaou — COMMENTED** [review]
@ventilooo The notification didn't display all the email details. It display details of email only if the email sending fails

**Review by rimbendhaou — APPROVED** [review]
Ok

**Comment by rimbendhaou** [comment]
:+1:

**Review by rimbendhaou — APPROVED** [review]
good

**Pull request merged**